packages/core/src/oas-types.ts

```diff
@@ -100,6 +103,10 @@ export function detectSpec(root: any): SpecVersion {
     throw new Error(`Unsupported AsyncAPI version: ${root.asyncapi}`);
   }
 
+  if (root.arazzo) {
+    return SpecVersion.Arazzo;
```

Comment:
Could we also check the version just in case? 
Like this: `root.asyncapi.startsWith('2.'))`